Tests/bytesTests/Dictionary+DiffTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Dictionary+DiffTests.swift
+//
+//
+//  Created by Thorsten Stark on 27.05.22.
+//
+
+import bytes
+import XCTest
+
+class ContentPageTests: XCTestCase {
```

Comment:
> Isn't it quite obsolete to write tests for a function, that could be deprecated, since swift 5.1 (iOS 13) introduced `difference(from:by:)`https://developer.apple.com/documentation/swift/array/3200716-difference

Answer to my self 😅: Not really, since it can still be used, even though it would be deprecated.

Reply:
Another reason would be that the `difference` function you found works with Arrays, and here we have Dictionaries ;)

Reply:
Right, diffing is only available for ordered collection types, which excludes Dictionaries 🙈